loc/de_de.json

```
@@ -446,10 +383,15 @@
     "details_del_wb_q": "Dieses Wallet enthält noch bitcoin. Ohne vorhandenes Backup der mnemonische Phrase sind diese unwiederbringlich verloren. Um ein versehentliches entfernen zu vermeiden, gib bitte das Wallet-Guthaben von {balance} satoshis ein.",
     "details_delete": "Löschen",
     "details_delete_wallet": "Wallet löschen",
+    "details_derivation_path": "Ableitungspfad",
     "details_display": "In Wallet-List anzeigen",
     "details_export_backup": "Exportieren / Backup",
     "details_marketplace": "Marktplatz",
     "details_master_fingerprint": "Fingerabdruckerkennung",
+    "details_ms_l": "{m} von {n} Altformat (p2sh)",
+    "details_ms_ns": "{m} von {n} native segwit (p2wsh)",
+    "details_ms_ws": "{m} von {n} wrapped segwit (p2sh-p2wsh)",
+    "details_multisig_type": "mehrfachsignatur",
```

Nouns are always uppercase in Germn
```suggestion
    "details_multisig_type": "Mehrfachsignatur",
```